src/ompl/tools/thunder/src/Thunder.cpp

```diff
@@ -131,10 +136,10 @@ void ompl::tools::Thunder::setup()
             experienceDB_->getSPARSdb()->setProblemDefinition(pdef_);
             experienceDB_->getSPARSdb()->setup();
 
-            experienceDB_->getSPARSdb()->setStretchFactor(1.6); 
-            experienceDB_->getSPARSdb()->setSparseDeltaFraction(
-                0.1);  // was 0.05 // vertex visibility range  = maximum_extent * this_fraction
-            experienceDB_->getSPARSdb()->setDenseDeltaFraction(0.001);
+            experienceDB_->getSPARSdb()->setStretchFactor(getStretchFactor());
+            experienceDB_->getSPARSdb()->setSparseDeltaFraction(getSparseDelta());  // was 0.05 // vertex visibility range  =
+                                                                       // maximum_extent * this_fraction
+            experienceDB_->getSPARSdb()->setDenseDeltaFraction(getDenseDelta());
```

Comment:
Are these inside the class methods? You don't need to call "get" functions to access private variables. Simply have 
`experienceDB_->getSPARSdb()->setStretchFactor(stretch_factor_)`